mimic/rest/identity_admin_api.py

```diff
@@ -0,0 +1,102 @@
+# -*- test-case-name: mimic.test.test_identity_admin -*-
+"""
+Mocks for the identity admin API.
+"""
+from json import dumps, loads
+
+from zope.interface import implementer
+from twisted.plugin import IPlugin
+
+from mimic.imimic import IAPIMock
+from mimic.rest.mimicapp import MimicApp
+from mimic.model.identity_admin_objects import bad_request
+
+
+@implementer(IAPIMock, IPlugin)
```

Comment:
@glyph Q: Might it be useful to rename mimic.rest.auth_api to mimic.rest.identity_api? Especially as it takes on more non-auth functionality like the identity-admin functionality?